.github/CONTRIBUTING.md

```diff
@@ -31,6 +31,19 @@ a proposal for your work first, to be sure that we can use it.
 Before you submit an issue, please search the issue tracker, maybe an issue for your identified problem already exists and you can join the discussion there only.
 For bugs, please try to provide as much information possible to reproduce it as possible, that will really help us a lot!
 
+## Report bugs using Github's [issues](https://github.com/UdacityMobileWebScholarship/showcase-app/issues)
```

Comment:
@optimistanoop There is no need of this as it is already mentioned how to report a bug. Please remove this.